tests/unit/core/test_sdk_api.py

```diff
@@ -25,6 +25,11 @@ def test_object_should_be_instance_of_class(self):
         sdk_api = SdkApi(self.basic_props, self.default_url)
         self.assertIsInstance(sdk_api, SdkApi)
 
+    def test_object_should_be_instance_with_logger_set_to_false(self):
```

Comment:
Need some suggestion on the unit test :relaxed: 
When I was testing with `log=False` the logger is not returning `None` (I guess maybe it should not)

Reply:
Hi @aadityasinha-dotcom,

I believe the logger is still present, but the Logger will have its `disabled` class variable set to `true` if logging is turned off:
https://github.com/zowe/zowe-client-python-sdk/blob/eb1ad71abc2de293d42e8bd1c34cf2fb42b80abb/src/core/zowe/core_for_zowe_sdk/logger.py#L90-L100

Reply:
I see, thank you for the help :+1: